style/js/shield_defs.js

```diff
@@ -559,6 +559,52 @@ export function loadShields(shieldImages) {
   shields["US:SC:Business"] = banneredShield(shields["US:SC"], ["BUS"]);
   shields["US:SC:Alternate"] = banneredShield(shields["US:SC"], ["ALT"]);
 
+  shields["US:TX"] = roundedRectShield("white", "black", "black", 1, 1, null);
+  shields["US:TX:Loop"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["LOOP"]
+  );
+  shields["US:TX:Spur"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["SPUR"]
+  );
+  shields["US:TX:Business"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["BUS"]
+  );
+  shields["US:TX:Park"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["PARK"]
+  );
+  shields["US:TX:Beltway"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["BELT"]
+  );
+  shields["US:TX:FM"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["F.M."]
+  );
```

Comment:
There are a few Business Farm-to-Market Roads, which are tagged [`network=US:TX:FM:Business`](https://taginfo.openstreetmap.org/tags/network=US%3ATX%3AFM%3ABusiness). Use `banneredShield()` to give it a “BUS” banner on top of the “F.M.” banner.